lib/coxir/struct/invite.ex

```diff
@@ -17,9 +17,18 @@ defmodule Coxir.Struct.Invite do
 
   Returns an invite object upon success
   or a map containing error information.
+
+  Accepts an optional argument `with_counts`
+  in which the invite struct will contain two
+  extra keys, approximate_member_count, the
+  number of members in the server, and
+  approximate_presence_count which is the number
+  of users in the server who are currently online.
   """
-  def get(code) do
-    API.request(:get, "invites/#{code}")
+  @spec get(invite, boolean()) :: map
+
+  def get(code, with_counts \\ false) do
```

Comment:
Can the param be renamed to `with_counts?`?

Reply:
Addressed in eb1725b133487b2dee9643ee062104a390a0b36f